app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The idea is using a ConstraintLayout as a main Layout not inside FrameLayout. Please, change this

Reply:
✔ **Done**